README.md

```diff
@@ -2,7 +2,7 @@
 
 
 > this the html and css portifolio project.
-
+this is the [link](https://divinecharlotte.github.io/portifolio-setup/) to my portifolio project. 
```

Comment:
- [ ] Please use a separate heading for the live version and put the link inside that heading. For now, it will be hard for users to find the live version link in the project description.